### PR TITLE
Upgrade ember-lodash dependency

### PR DESCRIPTION
**Pull request by pixelhandler**
With ember-cli-raygun and ember-cli-mirage 0.2.8 I had an error in the mirage serializer code `'TypeError: (0 , _lodash.default)(...).map(...).flatten(...).compact(...).uniqBy is not a function`

There was a conflict with lodash. Updating ember-lodash resolved the conflict.

### CODE REVIEW

**Comment by nikz** [comment]
Looks nice - and a `yarn.lock` to boot. Thanks!

**Comment by pixelhandler** [comment]
@nikz The yarn.lock should only matter for development of this addon, A consuming application will have it's own yarn.lock file. It's up to you I could remove it.

The main thing was the dependency for ember-lodash needed an upgrade as the older version can cause conflict with other addons, in my case with ember-cli-mirage.

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
   "dependencies": {
     "ember-cli-babel": "^5.1.6",
     "chalk": "^1.1.3",
-    "ember-lodash": "^0.0.10",
```
Comment (pixelhandler) [inline package.json]: Version `^0.0.10` will clash with other addons defining the same dependency with a current release, e.g. `^4.0`. The thing about `dependencies` in ember addons is that the last one seems to win. In my case  ember-cli-raygun's dependency for ember-lodash beat out ember-cli-mirage's dependency.

**Comment by pixelhandler** [comment]
@nikz would you be open to publishing a v1.2.1 release after the ember-lodash upgrade merges?

**Comment by nikz** [comment]
👍 yup, all very good - thanks.

`1.2.1` is now live - I've checked npm this time ;)

**Comment by pixelhandler** [comment]
@nikz thanks I see 1.2.1 on NPM,  https://www.npmjs.com/package/ember-cli-raygun

How about a 1.2.1 tag/release here in the repo as well for b2329a4

**Comment by nikz** [comment]
Didn't push tags, my bad

**Pull request merged**